fwutil/lib.py

```diff
@@ -344,18 +348,15 @@ def __parse_component_section(self, section, component, is_module_component=Fals
                 self.__module_component_map[section][key1] = OrderedDict()
 
             if value1:
-                if len(value1) != 3:
+                if len(value1) != 1 and len(value1) != 2:
```

Comment:
can you elaborate what this change stands for? it will be nice if that will be documented in the code as well.

Reply:
@liat-grozovik this is needed to make `version` field in `platform_components.json` optional.